genestack_client/genestack_shell.py

```diff
@@ -144,6 +144,17 @@ def get_short_description(self):
     def run(self):
         """
         Override this method to implement the command action.
+
+        Return value of this method is always ignored.
+        Exception thrown from this method means abnormal termination of the command.
+
+        If this command is executed not in shell mode,
```

Comment:
`If this command is executed in the shell mode, the failed state is ignored, otherwise exit code ``1`` is returned.